docs/workflow/README.md

```diff
@@ -12,81 +18,72 @@ The repo can be built for the following platforms, using the provided setup and
-```
+## Getting Yourself Started
 
-For Windows:
-```cmd
-build.cmd
-```
+The runtime repo can be built from a regular, non-administrator command prompt, from the root of the repo.
 
-This builds the product (in the default debug configuration), but not the tests.
+The repository currently consists of three different major parts:
 
-For information about the different options available, supply the argument `--help|-h` when invoking the build script:
-```
-build -h
-```
+* The Runtimes
+* The Libraries
+* The Installer
 
-On Unix like systems, arguments can be passed in with a single `-` or double hyphen `--`.
+More info on this, as well as the different build configurations in the [Configurations and Subsets section](#configurations-and-subsets).
 
-The repository currently consists of different major parts: the runtimes, the libraries, and the installer.
-To build just one part you use the root build script (build.cmd/sh), and you add the `-subset` flag.
+This was a concise introduction and now it's time to show the specifics of building specific subsets in any given supported platform, since most likely you will want to customize your builds according to what component(s) you're working on, as well as how you configured your build environment. We have links to instructions depending on your needs [in this section](#full-instructions-on-building-and-testing-the-runtime-repo).
 
-## Editing and Debugging
+* For instructions on how to edit code and make changes, see [Editing and Debugging](editing-and-debugging.md).
+* For instructions on how to debug CoreCLR, see [Debugging CoreCLR](/docs/workflow/debugging/coreclr/debugging-coreclr.md).
+* For instructions on using GitHub Codespaces, see [Codespaces](/docs/workflow/Codespaces.md).
 
-For instructions on how to edit code and debug your changes, see [Editing and Debugging](editing-and-debugging.md).
-
-For instructions on using GitHub Codespaces, see [Codespaces](Codespaces.md).
-
-## Configurations
+## Configurations and Subsets
 
 You may need to build the tree in a combination of configurations. This section explains why.
 
-A quick reminder of some concepts -- see the [glossary](../project/glossary.md) for more on these:
+<!-- LINK-UPDATES -->
+A quick reminder of some concepts -- see the [glossary](/docs/project/glossary.md) for more on these:
 
 * **Debug Configuration** -- Non-optimized code.  Asserts are enabled.
-* **Checked Configuration** -- Optimized code. Asserts are enabled.  Only relevant to CoreCLR runtime.
-* **Release Configuration** -- Optimized code. Asserts are disabled. Runs at the best speed, and suitable for performance profiling. You will have limited debugging experience.
+* **Checked Configuration** -- Optimized code. Asserts are enabled.  _Only relevant to CoreCLR runtime._
+* **Release Configuration** -- Optimized code. Asserts are disabled. Runs at the best speed, and suitable for performance profiling. You will have limited debugging experience with this one.
```

Comment:
```suggestion
* **Release Configuration** -- Optimized code. Asserts are disabled. Runs at the best speed, and suitable for performance profiling. This will impact the debugging experience due to compiler optimizations and make understanding what the debugging is showing difficult to reason about relative to source code.
```

It isn't "limited" rather it is degraded. I am being pedantic here, but my point is you can debug just fine and all the tools will work as designed. The issue is the compiler has optimized the code to a point where debugging may be more difficult.